src/OnboardingSPA/pages/Steps/Ecommerce/stylesheet.scss

```diff
@@ -118,12 +205,22 @@ div.onboarding-product-step {
 	}
 
 	.step-product-numbers {
-		display: flex;
-		width: 100%;
+		display: grid;
 		align-items: center;
+		grid-auto-flow: row;
```

Comment:
Receiving certain warnings due to grid-auto-flow @aulisius 
![image](https://user-images.githubusercontent.com/53893563/190088911-3365ccac-485b-456d-965f-98d15648bf2a.png)

Reply:
Fixed the warnings